pipeline/beam_tables.py

```diff
@@ -622,12 +622,6 @@ def run_beam_pipeline(self, scan_type: str, incremental_load: bool,
       # PCollection[Tuple[filename,line]]
       lines = _read_scan_text(p, new_filenames)
 
-      # Reshuffle to break fusion on long files
```

Comment:
Do you wanna comment it out instead and leave a comment?

Reply:
I think the better place to record it as a possibility is in the pull requests adding/removing it, not as a lingering code comment.